courses/free-courses-en.md

```diff
@@ -1401,6 +1401,7 @@
 * [Learn Ruby](https://www.learnrubyonline.org)
 * [RESTful API with Ruby On Rails 5](https://www.udemy.com/course/restful-api-with-ruby-on-rails-5/) - Udemy
 * [Ruby on Rails a Beginners Guide](https://www.udemy.com/course/ruby-on-rails-a-beginners-guide-free) - Stephen Chesnowitz (Udemy)
+* [Ruby Programming Language - Full Course](https://youtu.be/t_ispmWmdjY) - Mike Dane (Giraffe Academy)
```

Comment:
We don't allow shortened links.
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes.
when you've made changes, please add `(changes made)` to the title of your PR